olive/model/__init__.py

```diff
@@ -577,6 +577,18 @@ def load_model(self, rank: int = None) -> torch.nn.Module:
 
         return model
 
+    def loads_model_from_hf_config(self) -> bool:
```

Comment:
the name seems like a load behavior but it is used as a flag. better to change the name

Reply:
I used `loads` as an adjective and not verb. Maybe something like `model_loaded_from_hf_config` is clearer?

Reply:
can_model_load_from_hf_config?

Reply:
Done! Changed to `is_model_loaded_from_hf_config`